app/views/speeches/show.html.erb

```diff
@@ -1,6 +0,0 @@
-<% page_title edition_page_title(@document), t_display_type(@document, 10) %>
-<%= render  template: 'documents/show',
-            locals: {
-              header_title: t_display_type(@document),
-              show_share_links: true
```

Comment:
At what point can the `show_share_links: true` part of the `show` template be removed? I think News articles, Speeches and Consultations were the only uses and they've all been migrated?